lumeer-core/src/main/java/io/lumeer/core/facade/DocumentFacade.java

```diff
@@ -169,6 +171,8 @@ private void updateCollectionMetadata(Collection collection, Set<String> attribu
          }
 
       });
+      System.out.println("LMRDF adding " + attributesToInc.toString());
```

Comment:
These debug outputs are probably not needed anymore.

Reply:
Oh yes, I will delete it.